torchgeo/samplers/utils.py

```diff
@@ -55,17 +55,16 @@ def get_random_bounding_box(
     """
     t_size = _to_tuple(size)
 
-    width = (bounds.maxx - bounds.minx - t_size[1]) // res
-    height = (bounds.maxy - bounds.miny - t_size[0]) // res
+    # May be negative if bounding box is smaller than patch size
+    width = (bounds.maxx - bounds.minx - t_size[1]) / res
+    height = (bounds.maxy - bounds.miny - t_size[0]) / res
 
     minx = bounds.minx
     miny = bounds.miny
 
-    # random.randrange crashes for inputs <= 0
```

Comment:
This is a dead comment, we no longer use `random.randrange` and the input can no longer be negative